### ISSUE TITLE
Logs improvement

### ISSUE THREAD

**Issue by ArturWieczorek** [input-output-hk/smash#79]
I was thinking about this snippet:

```
[smash-node.fetch:Info:45] [2020-09-09 12:27:31.50 UTC] Pools without offline metadata: 1
[smash-node.fetch:Info:45] [2020-09-09 12:27:31.53 UTC] Request: https://raw.githubusercontent.com/aaa/a/master/a.json
[smash-node.fetch:Warning:45] [2020-09-09 12:27:32.06 UTC] HTTP Response : 404
[smash-node.fetch:Info:45] [2020-09-09 12:27:32.07 UTC] fetchLoop: 0 runnable, 1 pending
```

Should not request `Request: https://raw.githubusercontent.com/aaa/a/master/a.json` be also informing about pool id that it is associated with ? Otherwise what useful info it gives me that _some_ request for _some_ pool is returning 404 ?

If I had a pool id it would be easier for me to query:
http://localhost:3100/api/v1/errors?poolId=poolid123...

On the other hand what if there many pools with errors - would not that just clutter the logs without being really helpful ?
In short - I am not sure if those errors in logs are helpful/needed or they should stay there but contain pool id so you can easier query endpoint with particular poolId `?poolId=poolid123...`

**Comment by ksaric** [comment]
Closing, implemented in https://github.com/input-output-hk/smash/pull/94
